docker_utils/get_container.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 SD Elements Inc.
+#
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of SD Elements Incorporated and its suppliers,
+# if any.  The intellectual and technical concepts contained
+# herein are proprietary to SD Elements Incorporated
+# and its suppliers and may be covered by U.S., Canadian and other Patents,
+# patents in process, and are protected by trade secret or copyright law.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from SD Elements Inc..
+# Version
+
+version='0.1'
+
+# Import the standard shell library
+# shellcheck source=../shtdlib.sh disable=SC1091
+source "$(dirname "${0}")/../shtdlib.sh" &> /dev/null || ../shtdlib.sh &> /dev/null || source ./shtdlib.sh &> /dev/null || source shtdlib.sh
+
+# A helper function to ensure containers exist locally before launch
+function get_container {
+    debug 10 "get_container called with: ${*}"
+    args=( "${@}" )
+    if [ "${#args[@]}" -lt 3 ] ; then
+        color_echo red "${0} needs at least three arguments, none were provided"
```

Comment:
Would be good too specify which arguments should be required in this output, or add some optargs to specify each argument.

Reply:
Done and done.